RoboJackets/robocup-software
# Adding Seeking

## Description
Adds seeking functionality to offense which makes idle robots try to 'get open'. 

## Associated / Resolved Issue
Resolves https://app.clickup.com/t/86aytmv75

## Steps to Test
1. make run-sim
2. Move opposition robots between the ball and our team robot

**Expected result:** Idle robots should be moving around the field to get open

## Key Files to Review
 * offense.cpp


## Review Checklist


## (Optional) Sub-issues (for drafts)
_Note: if you find yourself breaking this PR into many smaller features, it may make sense to break up the PR into logical units based on these features._
